tests/views/test_respondents.py

```diff
@@ -23,6 +24,9 @@
 iac_1 = 'jkbvyklkwj88'
 iac_2 = 'ljbgg3kgstr4'
 
+get_respondent_root = f'http://localhost:8085/party-api/v1/respondents/'
```

Comment:
did you mean to leave `localhost:8085` hard-coded in here?

Reply:
Yeah, it's not a problem in this context.